stubs/stripe/stripe/api_resources/application_fee_refund.pyi

```diff
@@ -1,12 +1,13 @@
-from typing import Any
+from _typeshed import Self
+from typing import Any, NoReturn
 
 from stripe.api_resources import ApplicationFee as ApplicationFee
 from stripe.api_resources.abstract import UpdateableAPIResource as UpdateableAPIResource
 
 class ApplicationFeeRefund(UpdateableAPIResource):
     OBJECT_NAME: str
     @classmethod
-    def modify(cls, fee, sid, **params): ...
-    def instance_url(self): ...
+    def modify(cls: type[Self], fee, sid: str, **params) -> Self: ...  # type: ignore[override]
```

Comment:
And here

Reply:
this class inherits from the updatable api resource so it has the same behavior as Customer

```python
>>> customer = stripe.Customer.create(); type(customer.modify(customer.id, name="foo"))
<class 'stripe.api_resources.customer.Customer'>
```